CHANGELOG.md

```diff
@@ -10,11 +10,11 @@
 * [BUGFIX] OTLP: Don't generate target_info unless at least one identifying label is defined. #13991
 * [BUGFIX] OTLP: Don't generate target_info unless there are metrics. #13991
 
-## 2.52.0-rc.1 / 2024-05-03
+## 2.52.1 / 2024-05-29
 
-* [BUGFIX] API: Fix missing comma during JSON encoding of API results. #14047
```

Comment:
This is in 2.52.0 and already on main, I'd add it on line 45 (end of 2.52.0 changelog)

Reply:
I wouldn't add this to 2.52.0 as a BUGFIX because it didn't exist before 2.52.0. It comes with a change made in https://github.com/prometheus/prometheus/pull/13744, then the bug was discovered and fixed in the release candidate.

In other words, users who did not use the release candidate would never have crossed paths with this bug, and that's why I thought we shouldn't mention it in the changelog